examples/gno.land/p/gov/dao/types.gno

```diff
@@ -0,0 +1,43 @@
+package dao
+
+import (
+	"std"
+
+	"gno.land/p/gov/proposal"
+)
+
+type VoteOption string
+
+const (
+	YesVote VoteOption = "YES"
+	NoVote  VoteOption = "NO"
+)
+
+func (v VoteOption) String() string {
+	return string(v)
+}
+
+// Vote encompasses a single GOVDAO vote
+type Vote struct {
+	Address std.Address // the address of the voter
+	Option  VoteOption  // the voting option
+}
+
+// DAO defines the DAO abstraction
+type DAO interface {
```

Comment:
move to dao.gno

Reply:
Moved:

[ae0639a](https://github.com/gnolang/gno/pull/2581/commits/ae0639a304fbc48cd4404cc968492943fb9dcb28)